content/blog/sprint-review-2211.md

```diff
@@ -0,0 +1,40 @@
+Title: Wazo Platform 22.11 Released
+Date: 2022-08-03T15:56:00
+Author: The Wazo Authors
+Category: Wazo Platform
+Tags: wazo-platform, development
+Slug: release-review-2211
+Status: published
+
+Hello Wazo Platform community!
+
+Here is a short review of the Wazo Platform 22.11 release.
+
+## Technical Features
+- **sysconfd**: The system configuration service has been rewritten to remove
+  unmaintained part of code
+
+## Ongoing Features
+- **Scalability**: Consume events from the bus instead of the Asterisk
+  websocket. This will improve the performance of Wazo and allow greater
+  scalability.
+
```

Comment:
- **Documentation**: Publish documentation describing bus events used by the Wazo Platform